### PR TITLE
Chore/update readme

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
Updates README to show new state of the product + reference the GitHub app. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [x] Documentation update  
- [x] Chore: cleanup/renaming, etc  
- [ ] RFC  

## Acceptance
- [x] I have read [the Contributing guidelines](CONTRIBUTING.md)
- [x] I have read the [Code of Conduct](CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,49 +10,31 @@
 [![Rating](https://img.shields.io/visual-studio-marketplace/r/WatermelonTools.watermelon-tools?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=WatermelonTools.watermelon-tools&ssr=false#review-details)
 [![Discord](https://img.shields.io/discord/933846506438541492?style=flat-square)](https://discord.com/invite/H4AE6b9442)
 
-Watermelon is your **Open Source** Code Archeology Toolbox for **Visual Studio Code**. We allow developers to find the most relevant piece of context for a given block of code. We help you __grok__ codebases by indexing information from git, ticketing, and messaging systems. 
+**Watermelon helps your team merge Pull Requests With Business Value**.
```
Comment (EstebanDalelR) [inline README.md:13]: The extension works for GitLab too, is this really fo merging PRs?

**Review by EstebanDalelR — COMMENTED** [review]
Only one title to check

File: README.md [hunk 2]
```diff
@@ -10,49 +10,31 @@
...
+## Integrations
 
-If you want to do a deeper research about the code context history, simply click such button. 
+We currently support the following integrations
 
-### Get Full Code Context
+|               | Git                                   | Project Management | Messaging | Documentation |
+| :------------ | :------------------------------------ | :----------------- | :-------- | :-----------  |
+| VS Code       | GitHub, GitLab, Bitbucket             | Jira               | Slack     |               |
+| GitHub App    | GitHub                                | Jira               | Slack     | Notion        |
```
Comment (EstebanDalelR) [inline README.md:29]: The extension currently does not fetch from Notion

Comment (baristaGeek) [reply README.md:29]: Correct. This is how the rendered table looks

Screen Shot 2023-06-15 at 5 20 16 PM

**Review by EstebanDalelR — APPROVED** [review]

**Pull request merged**